### PR TITLE
added meteor version 2.9.1

### PR DESCRIPTION
**Pull request by gerwinbrunner**
Hi, since meteor 2.9.1 is out, I forked this repo and run:
`./update.sh --meteor-version 2.9.1 --node-version 14.21.2`

Finally, I created this PR

### CODE REVIEW

File: .github/workflows/continuous-integration-workflow.yml [hunk 1]
```diff
@@ -55,6 +55,7 @@ jobs:
           - '2.7.3'
           - '2.8.0'
           - '2.8.1'
+          - '2.9.1'
```
Comment (diavrank) [inline .github/workflows/continuous-integration-workflow.yml:58]: add 2.9.0
```suggestion
          - '2.9.0'
          - '2.9.1'
```

File: versions.sh [hunk 2]
```diff
@@ -44,5 +44,6 @@ meteor_versions=( \
 	'2.7.2' \
 	'2.7.3' \
 	'2.8.0' \
-	'2.8.1'
+	'2.8.1' \
+	'2.9.1'
```
Comment (diavrank) [inline versions.sh:48]:
```suggestion
	'2.9.0'
	'2.9.1'
```

File: support.sh [hunk 3]
```diff
@@ -54,5 +54,6 @@ set_node_version() {
 	elif [[ "$1" == 2.7.3 ]]; then node_version='14.19.3'
 	elif [[ "$1" == 2.8.0 ]]; then node_version='14.20.1'
 	elif [[ "$1" == 2.8.1 ]]; then node_version='14.21.1'
+	elif [[ "$1" == 2.9.1 ]]; then node_version='14.21.2'
```
Comment (diavrank) [inline support.sh:57]:
```suggestion
	elif [[ "$1" == 2.9.0 ]]; then node_version='14.21.2'
	elif [[ "$1" == 2.9.1 ]]; then node_version='14.21.2'
```

**Review by diavrank — CHANGES_REQUESTED** [review]
I suggest you run first the script with version 2.9.0, make a commit and after run the script with 2.9.1. This is to include all the Meteor versions.

**Comment by GeoffreyBooth** [comment]
> I suggest you run first the script with version 2.9.0, make a commit and after run the script with 2.9.1. This is to include all the Meteor versions.

Yes, please include 2.9.0 as well.

**Comment by diavrank** [comment]
@GeoffreyBooth I've created a PR with both versions

**Comment by GeoffreyBooth** [comment]
Closed by #132.

**Pull request closed without merging**